NuRadioReco/modules/io/event_parser_factory.py

```diff
@@ -98,10 +98,16 @@ def scan_files_2_2(self, iF, current_byte):
                 is_generic_detector = False
             else:
                 is_generic_detector = detector_dict['generic_detector']
+
+            if 'detector_parameters' not in detector_dict.keys():
```

Comment:
no`.keys()` needed

Reply:
actually you can use `dict.get(`detector_parameters`, {})` here as well

Reply:
I copied that syntax from the generic_detector code just above. I'll change that as well then ;)